src/components/MenuCategory/MenuCategory.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { MenuItem } from "../../components";
+
+export class MenuCategory extends React.Component {
+  render() {
+    const { title, items, breads, addToOrder } = this.props;
+
+    return (
+      <React.Fragment>
+        <h2>{title}</h2>
+        <ul className="menu__category">
```

Comment:
It might be best to have this as its own element `menu-category` as it is its own component, rather than a child element of `menu__` 👍

Reply:
I always find BEM naming hard for stuff like this… my thinking was `menu__category` will always be a child of `menu` - I named `menu-item` as a component in it's own right - what are your thoughts?

Reply:
I guess in this case it will be fine 👍 its not a library of components I guess so can be coupled to `Menu` as always being a child